Note that `fee_ref` will be removed if XRPFees is enabled (#2183)
XRPLF/xrpl-dev-portal: content/references/http-websocket-apis/public-api-methods/subscription-methods/subscribe.md

|:--------------------|:--------------------------|:---------------------------|
| `type`              | String                    | `ledgerClosed` indicates this is from the ledger stream |
| `fee_base`          | Number                    | The [reference transaction cost](transaction-cost.html#reference-transaction-cost) as of this ledger version, in [drops of XRP][]. If this ledger version includes a [SetFee pseudo-transaction](setfee.html) the new transaction cost applies starting with the following ledger version. |

> | `fee_ref`           | Number                    | _(May be omitted)_ The [reference transaction cost](transaction-cost.html#reference-transaction-cost) in "fee units". If the _[XRPFees amendment][]_ is enabled, it is permanently omitted, as it will no longer be relevant. |

Reviewer: ```suggestion
| `fee_ref`           | Number                    | _(May be omitted)_ The [reference transaction cost](transaction-cost.html#reference-transaction-cost) in "fee units". If the _[XRPFees amendment][]_ is enabled, this field is permanently omitted as it will no longer be relevant. |
```

> | `fee_ref`           | Number                    | _(May be omitted)_ The [reference transaction cost](transaction-cost.html#reference-transaction-cost) in "fee units".  Will be permanently omitted if the _[XRPFees amendment][]_ is enabled, because it will no longer have any meaning. |

Reviewer 2: ```suggestion
| `fee_ref`           | Number                    | _(May be omitted)_ The [reference transaction cost](transaction-cost.html#reference-transaction-cost) in "fee units". If the _[XRPFees amendment][]_ is enabled, it is permanently omitted, as it will no longer be relevant. |
```
